Makefile

```diff
@@ -105,7 +105,7 @@ test/unit-tests:
 ## Generate README.md with Terradoc
 .PHONY: terradoc
 terradoc:
-	$(call quiet-command,terradoc -o README.md README.tfdoc.hcl)
+	$(call quiet-command,terradoc generate -o README.md README.tfdoc.hcl)
```

Comment:
@mariux I've made myself a Shortcut task to update this in the other repo's.

Reply:
Mabe checkout https://github.com/gruntwork-io/git-xargs for automating such tasks across several repositories :)

Reply:
@soerenmartius Perfect thank you! Was gonna ask if there was a tool to make this easier